tests/testthat/test-lancelot-vaccination.R

```diff
@@ -3185,21 +3200,27 @@ test_that("Vaccine skip works in all compartments", {
   y <- mod$simulate(1:200 * 4)
   y <- mod$transform_variables(drop(y))
 
-  ## cum_n_X_vacc_skip should be equal to the totals in vaccine class 5
+  ## cum_n_X_vacc_skip[, 3, , ] should be equal to the totals in vaccine class 5
+  ## but there should be no other vacc skip moves
   expect_true(any(y$cum_n_S_vacc_skip > 0))
-  expect_equal(y$cum_n_S_vacc_skip,
+  expect_true(any(y$cum_n_S_vacc_skip[, c(1, 2, 4, 5), , ] == 0))
```

Comment:
should this be all rather than any? 
Same in all the ones below

Reply:
Fixed